app/assets/stylesheets/pages/case_contacts.scss

```diff
@@ -113,6 +113,10 @@ legend {
     cursor: pointer;
 }
 
+.remove-expense-button {
```

Comment:
Is this doable through Bootstrap code?

Reply:
@schoork I can look into that, haven't worked with bootstrap in a long time but I'll check it out